### ISSUE TITLE
LeftAxis and RightAxis number of labels and postion

### ISSUE THREAD

**Issue by adriamt** [PhilJay/MPAndroidChart#3442]
Hi, I have a LineChart with two different LineDataSets and I have added axis dependency to each LineDataSet, like this:
```java
setCho.setAxisDependency(YAxis.AxisDependency.LEFT);
setKcal.setAxisDependency(YAxis.AxisDependency.RIGHT);
```
My question is if there is any form to have the same number  of labels at the same position in each YAxis, like this picture:
![image](https://user-images.githubusercontent.com/11394100/30638544-0eaf51f4-9dfc-11e7-8872-ed461ff7f9a1.png)
In the image, you can see that on the left axis there is a granularity of 200 while on the right axis the granularity is of 300, but the number of labels are the same and are located at the same Y position.

Any clue how to accomplish this?

**Comment by davidgoli** [comment]
Have you tried:

```java
chart.getAxisLeft().setGranularity(200f);
chart.getAxisRight().setGranularity(300f);
```

**Comment by adriamt** [comment]
@davidgoli Yes, I've tried, but the labels aren't at the same position. I want that the Y value of 200 is at the same Y position that the Y value for 300 for the other axis.

**Comment by lqq1312** [comment]
@adriamt hello,I want to acheive your this char Xaxis effect.Can you give me some advises,thank you?
